engine/src/mblandroidfont.cpp

```diff
@@ -457,9 +457,8 @@ static bool create_custom_font_from_path(MCStringRef p_path, FT_Library p_librar
         for (uint32_t i = 0; i < FT_Get_Sfnt_Name_Count(t_font_face); i++)
         {
             // Attempt to fetch the name of the font.  The name is name id 4 as defined in https://developer.apple.com/fonts/TTRefMan/RM06/Chap6name.html
-            // It appears that the platform to use here is 1 (Macintosh according to the spec).
             FT_Get_Sfnt_Name(t_font_face, i, &t_sft_name);
-            if (t_sft_name.name_id == 4 && t_sft_name.platform_id == 1 && t_sft_name.encoding_id == 0 && t_sft_name.language_id == 0 && t_sft_name.string_len != 0)
+            if (t_sft_name.name_id == 4 && t_sft_name.string_len != 0)
```

Comment:
Looking at the bug report I'm not sure this is correct.

There are two issues here from what I can see...

Several of the fonts listed in the bug report have their names encoded as UTF-16BE and not native - that is probably why some of those (which do have matching criteria!) do not work correctly (I also suspect they are incorrectly built fonts!).

The second is that we are not searching will less strict criteria if stricter criteria fail.

The first of these is more serious than the second as I suspect it is why several of the fonts in that bug report which do have 4100 entries are not working - and means that even if other (less strict) searches are done, it still won't work.